homework_1/zad1/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/pkg/errors"
+	"log"
+	"strings"
+
+	"code-cadets-2021/homework_1/zad1/fizzbuzz"
+)
+
+func main() {
+	var start, end int
+
+	flag.IntVar(&start, "start", 1, "Starting value for game Fizzbuzz")
+	flag.IntVar(&end, "end", 10, "Ending value for game Fizzbuzz")
+
+	flag.Parse()
+
+	result, err := fizzbuzz.PlayFizzBuzz(start, end)
```

Comment:
`PlayFizzBuzz` is not too informative - it sounds like a function that does some [side effects](https://softwareengineering.stackexchange.com/questions/40297/what-is-a-side-effect), while it's actually a [pure function](https://en.wikipedia.org/wiki/Pure_function) that gets, calculates or generates FizzBuzz output. Please rename accordingly.